labrad/node/__init__.py

```diff
@@ -511,7 +501,7 @@ def __init__(self, nodename, host, port, credential):
         self.credential = credential
         self.servers = {}
         self.instances = {}
-        self.initMessages(True)
+        self.instances_to_notify = weakref.WeakSet()
```

Comment:
Added some comments. I want to get rid of `instances_to_notify` soon with some upcoming changes to `ServerProcess` itself. I've also included a note about this in the comments here.